### PR TITLE
Add web test for exportPythonPlain

### PR DESCRIPTION
**Pull request by rebornix**
Integration test for Export features on Web, it's now comparing against `main` yet as https://github.com/microsoft/vscode-jupyter/pull/10016 is not merged yet.

The test now covers:

* `exportPythonPlain` is invoked when exporting to Python.

### CODE REVIEW

File: src/test/datascience/export/fileConverter.vscode.web.test.ts [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+        when(configuration.getSettings(anything())).thenReturn(instance(settings));
+        when(settings.pythonExportMethod).thenReturn('direct');
+        when(filePicker.showDialog(anything(), anything(), anything())).thenReturn(
+            Promise.resolve(Uri.file('test.py'))
+        );
+        when(appShell.showErrorMessage(anything())).thenResolve();
+        when(exportPythonPlain.export(anything(), anything(), anything())).thenResolve();
+        when(filePicker.showDialog(anything(), anything())).thenResolve(Uri.file('foo'));
+        when(exportFileOpener.openFile(anything(), anything(), anything())).thenResolve();
+        fileConverter = new FileConverter(
```
Comment (rebornix) [inline src/test/datascience/export/fileConverter.vscode.web.test.ts:40]: The `FileConverter` on Web now differs from the Nodejs one. Not sure if it stays so after we migrate most features to Web, which depends on the implementation details of Export actions.

File: src/test/datascience/export/fileConverter.vscode.web.test.ts [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+            instance(exportPythonPlain),
+            instance(filePicker),
+            instance(exportFileOpener)
+        );
+    });
+    teardown(() => sinon.restore());
+
+    test('Export to Python is called when export method is Python', async () => {
```
Comment (IanMatthewHuff) [inline src/test/datascience/export/fileConverter.vscode.web.test.ts:48]: I know this is just the same thing as the existing vscode test. But looking at those tests, I'm not really sure they buy us much? In any case I have no issue with adding it.

Comment (IanMatthewHuff) [reply src/test/datascience/export/fileConverter.vscode.web.test.ts:48]: Just rethinking those existing tests myself. Feels like they are just confirming some internal implementation and that's it.

**Review by IanMatthewHuff — APPROVED** [review]
Approved, though I'm not sure our existing test was that useful :)

File: src/test/datascience/export/fileConverter.vscode.web.test.ts (hunk 2 above)
Comment (rchiodo) [reply src/test/datascience/export/fileConverter.vscode.web.test.ts:48]: Oh yeah this test is rather simple. I was hoping there was a test that opened a notebook and exported it, verifying the text got generated. 

This is just testing the internals of the fileConverter.

**Comment by rebornix** [comment]
Closing as there are too many changes happening since the PR was opened.

**Pull request closed without merging**